CMakeLists.txt

```diff
@@ -204,6 +204,8 @@ option(EXECUTORCH_BUILD_MPS "Build the MPS backend" OFF)
 
 option(EXECUTORCH_BUILD_NEURON "Build the backends/mediatek directory" OFF)
 
+option(EXECUTORCH_BUILD_OPENVINO "Build the Openvino backend" OFF)
```

Comment:
Can you add logging here https://github.com/pytorch/executorch/blob/254eecab290d37b64bec117d8a66025a9f6d6df8/build/Utils.cmake#L145 so it will be part of summary during build

Reply:
Thanks @cccclai, we added the logging.